bigquery/CHANGES.md

```diff
@@ -1,5 +1,7 @@
 ### 1.2.1 - 2021-XX-XX
 
+1.  Update all dependencies to latest versions.
```

Comment:
What do you think about making it a link to the PR? Since we're at it, why not.

Reply:
What is the syntax for linking PR in `md` file?